llvm/include/llvm/Object/COFFImportFile.h

```diff
@@ -91,6 +120,10 @@ struct COFFShortExport {
   /// file, this is "baz" in "EXPORTS\nfoo = bar == baz".
   std::string AliasTarget;
 
+  /// Specifies EXPORTAS name. In a .def file, this is "bar" in
+  /// "EXPORTS\nfoo EXPORTAS bar".
+  std::string ExportAs;
```

Comment:
This looks like an ABI break.

Reply:
If that's problematic, this specific part of the change isn't needed for Rust's scenario.

Reply:
If you can remove this part of the patch, that would be great.

Reply:
Done